x-pack/plugins/security_solution/public/common/components/top_n/index.tsx

```diff
@@ -15,12 +15,12 @@ import { InputsModelId } from '../../store/inputs/constants';
 import { useGlobalTime } from '../../containers/use_global_time';
 import type { BrowserFields } from '../../containers/source';
 import { useKibana } from '../../lib/kibana';
+import { combineQueries } from '../../lib/keury';
```

Comment:
Not to cause a bunch of file changes, but I believe it's `kuery`. Gotta love dev naming 😂

Reply:
I agree but did not touch it because it gets difficult for reviewers to see actual changes. But I will rename it.

Reply:
@michaelolo24 fixed. 🪛